kasa/cli.py

```diff
@@ -366,6 +366,26 @@ async def temperature(dev: SmartBulb, temperature: int, transition: int):
         return await dev.set_color_temp(temperature, transition=transition)
 
 
+@cli.command()
+@click.argument("effect", type=click.STRING, default=None, required=False)
+@click.pass_context
+@pass_dev
+async def effect(dev, ctx, effect):
+    """Set an effect."""
+    if not dev.has_effects:
+        click.echo("Device does not support effects")
+        return
+    if effect is None:
+        raise click.BadArgumentUsage(
+            "Setting an effect requires a named built-in effect.", ctx
+        )
```

Comment:
This should probably print the list of available effects instead of erroring out, otherwise there is no way for the user to find out which effects are available.

Reply:
08c258be5e76ebdd5f41762635f856ccad08c159